keyboards/bm60rgb_iso/rules.mk

```diff
@@ -0,0 +1,27 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
```

Comment:
I think this or console_enable will need to be set to no, looks like the firmware is too large to compile

Reply:
It compiles fine on my machine with 402 bytes free. (98% full). :)
It also passed the build check so i think it should be good.

Reply:
Console is generally not a particularly useful feature to have on production firmware, so it can be safely disabled, at least.

Reply:
I disabled the console and commands, now the firmware size got down from 98% to 77%.